src/home/rooms_sidebar.rs

```diff
@@ -20,7 +20,7 @@ live_design! {
 
     use crate::home::rooms_list::RoomsList;
 
-    pub RoomsSideBar = <AdaptiveView> {
+    pub RoomsSideBar = {{RoomsSideBar}} {
```

Comment:
this might be a merge conflict error, but we want the DSL override syntax here that I previously had. Please restore it.

```suggestion
    pub RoomsSideBar = {{RoomsSideBar}}<AdaptiveView> {
```